dimod/bqm/adjarraybqm.pxd

```diff
@@ -62,14 +62,39 @@ cdef class cyAdjArrayBQM:
             numerical offset. Note that when formed with SPIN-variables, biases
             on the diagonal are added to the offset.
 
-    The AdjArrayBQM is implemented using two c++ vectors. The first
-    vector contains the linear biases and the index of the start of each
-    variable's neighborhood in the second vector. The second
-    vector contains the neighboring variables and their associated quadratic
-    biases.
+    Notes:
 
-    The AdjArrayBQM is useful when the application requires fast iteration
-    over continuous memory.
+        The AdjArrayBQM is implemented using two c++ vectors. The first
+        vector contains the linear biases and the index of the start of each
+        variable's neighborhood in the second vector. The second
+        vector contains the neighboring variables and their associated quadratic
+        biases. The vectors, once initialized, are not resized.
+
+        Advantages:
+
+        - Very fast iteration over the biases
+
+        Disadvantages:
+        
+        - Does not support incremental construction
+        - Only supports float64 biases
+
```

Comment:
This is good for doc readers; if you worry it's too long for developers going through the code, you can also reduce it to 
```
* Advantages:  very fast iteration over the biases
* Disadvantages: does not support incremental construction; only supports float64 biases
* Intended Use: when performance is important and the BQM can be treated as read-only
```

Reply:
I am sort of imagining that these lists will grow over time, but it's a good point